### ISSUE TITLE
Copy/Paste From 1 Model View to Another Results in Duplicate Elements

### ISSUE THREAD

**Issue by audreykoz** [archimatetool/archi#482]
### Version of Archi, Operating System

I'm using Archi version 4.2.0 on a Mac running macOS Mojave Version 10.14.5
### Expected Behaviour

Copying/pasting elements from a view in one model to another view in a different model should only create 1 version of each element in the model tree.   
### Actual Behaviour

Elements referenced 2+ times in a view are duplicated when copy/pasted into a new view and model. 
### Steps to Reproduce the Behaviour
1. Create a view where an element is used 2+ times 
2. Copy all the elements in that view 
3. Paste the elements into a new view in a different model

**Comment by Phillipus** [comment]
Yes, the Copy/Paste code needs a bit of an overhaul at some point.

> I'm using Archi version 4.2.0 on a Mac running macOS Mojave Version 10.14.5

Latest version is 4.4 if you can get that. ;-)

**Comment by michaelansley** [comment]
What should the result be?

If I copy some part of a view to another view, it's not clear that *not* pasting the duplicates is the correct thing to do.  Is it?

**Comment by paugras1969** [comment]
> Yes, the Copy/Paste code needs a bit of an overhaul at some point.
> 
> I'm using Archi version 4.2.0 on a Mac running macOS Mojave Version 10.14.5
> 
> Latest version is 4.4 if you can get that. ;-)

hello , just tried that paste , paste special duplicate anyway in the target model. Feature request is still relevant . Thanks anyway for the great job done
